### PR TITLE
Feature/container fetch method renamed

### PR DESCRIPTION
**Pull request by xavier-rdo**

### CODE REVIEW

File: test/Container.spec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Container from '../src/Container';
+
+describe('Container', () => {
+    let container = new Container();
```
Comment (Tom32i) [inline test/Container.spec.js]:
```diff
-let
+const
```

**Comment by ogizanagi** [comment]
Just asking: as this is a package, and not an application, shouldn't you ignore the `yarn.lock` file?

File: test/Container.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    class MyClass { }
+    container.registerDefinition('my:class', MyClass);
+
+    describe('Container.get()', () => {
+        test('should return the expected service when asked for', () => {
+            expect(container.get('my:class') instanceof MyClass).toBeTruthy;
```
Comment (Tom32i) [inline test/Container.spec.js]: `expect(container.get('my:class')).toBeInstanceOf(MyClass)`

See doc: https://facebook.github.io/jest/docs/expect.html#tobeinstanceofclass

**Comment by chalasr** [comment]
Theoretically, this is a BC break. :trollface:

File: test/Container.spec.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        });
+
+        test('should throw an error if an unexisting service or param is accessed', () => {
+            expect(() => {
+                container.get('unexisting:service')
```
Comment (Tom32i) [inline test/Container.spec.js]: Missing `;` (or go inline).

File: package.json [hunk 4]
```diff
@@ -14,12 +14,17 @@
   },
   "homepage": "https://github.com/Elao/container.js#readme",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node_modules/.bin/jest"
```
Comment (Tom32i) [inline package.json]: Ici tu peux juste faire `jest` car on est dans package.json (il est intelligent)

**Comment by xavier-rdo** [comment]
- Removed yarn.lock file and added it to .gitignore
- Fixed Thomas' requirements
- Update package.json's script/test to run jest

**Pull request merged**